code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -301,7 +301,7 @@
 	operate_for(50, juicing = TRUE)
 	addtimer(CALLBACK(src, /obj/machinery/reagentgrinder/proc/mix_complete), 50)
 
-/obj/machinery/reagentgrinder/proc/mix_complete()
+/obj/machinery/reagentgrinder/proc/mix_complete() //what the fuck. why the fuck are mixed recipes hardcoded. seriously what in the actual shit. i mean its out of scope for my pr but hot damn
```

Comment:
tbh, unneeded rant comment

Reply:
but MOOOOOOOOOOOM, how else am i gonna squeeze as many line of code changes out of my PRs?

Reply:
i actually am ok with this comment thank you for ur opinions papa robo